test_vm/src/lib.rs

```diff
@@ -676,6 +686,28 @@ impl<'invocation, 'bs> InvocationCtx<'invocation, 'bs> {
         self.resolve_target(&self.msg.to).unwrap().1
     }
 
+    fn invoke_actor(&mut self) -> Result<RawBytes, ActorError> {
+        std::panic::catch_unwind(std::panic::AssertUnwindSafe(|| self.invoke())).unwrap_or_else(
+            |panic| {
+                if self.actor_exit.borrow().is_some() {
+                    let exit = self.actor_exit.take().unwrap();
+                    self.actor_exit.replace(None);
```

Comment:
`take` already leaves `None` in `actor_exit`
```suggestion
                    let exit = self.actor_exit.take().unwrap();
```

Reply:
ah, ok. thanks.

Reply:
done.